cemu/ui/codeeditor.py

```diff
@@ -16,12 +16,11 @@
 )
 
 import cemu.core
+from build.lib.cemu.const import DEFAULT_CODE_VIEW_FONT, DEFAULT_CODE_VIEW_FONT_SIZE
```

Comment:
what's `build.lib` here?

Reply:
I think PyCharm IDE optimized imports by mistake :) instead of import from cemu.const imported from different directory